### ISSUE TITLE
How to handle earphones?

### ISSUE THREAD

**Issue by udamken** [udamken/mindbell#5]
Original issue: https://github.com/mindful-marc/mindbell/issues/38
Original issue: https://github.com/mindful-marc/mindbell/issues/16

If tone is sent to earphones ... one user asked for possibility to choose a different ringtone only for earphone usage.

A different idea is to mute the bell when earphones are plugged in, optional of course.

**Comment by udamken** [comment]
Request came only once. Most requests are muting the bell when music is playing or pausing the music when bell sound is started. Both options have been implemented in https://github.com/udamken/mindbell/issues/3 and the bell can be muted manually.

Furthermore how should earphones be detected properly? Anything plugged in can be an earphone or a speaker. Same with bluetooth.

So there will be no more special earphone handling. Closed as wontfix.
